### ISSUE TITLE
About Our Grade

### ISSUE THREAD

**Issue by tieyong** [uchicago-mobi/MPCS51030-2019-Winter-Forum#159]
Are we going to get our final letter grade today?

**Comment by susanstevens** [comment]
Yes, I'll post the grades this evening.

**Comment by jarofmy** [comment]
Is there a way to view the grade we received on the final project, as well as any feedback?

**Comment by susanstevens** [comment]
Yes! I will email everyone with their final project feedback. You should receive an email from me sometime this week.
